copy-alerts/copy-alerts.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euxo pipefail
```

Comment:
i'd remove the -x, it really hammers stdout which can confuse folks

Reply:
removed -x .. Maybe yes but i thought more information is better but agreed too much verbosity.